packages/connect-web-bench/buf.gen.yaml

```diff
@@ -6,9 +6,9 @@ plugins:
   - name: connect-web
     out: src/gen/connectweb
     opt: ts_nocheck=false,target=ts
-  - remote: buf.build/protocolbuffers/plugins/js:v3.20.1-1
+  - plugin: buf.build/protocolbuffers/js
```

Comment:
We still need to pin the version though.

Reply:
Ah shoot.  We don't do this in protobuf-es either.  I'll update it there also.